simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch

```diff
@@ -1,10 +1,10 @@
-<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->                                                                                                            
+<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->
 <launch>
   <!-- Arguments -->
-  <arg name="model" default="$(optenv TURTLEBOT3_MODEL waffle_pi)" doc="model type [burger, waffle, waffle_pi]"/>                                                                                
-  <arg name="map_file" default="$(find cloudwatch_simulation)/maps/map.yaml"/>
-  <arg name="open_rviz" default="true"/>
-  <arg name="initial_pose_x" default="0.0"/>                                                                                                                                        
+  <arg name="model" default="$(optenv TURTLEBOT3_MODEL waffle_pi)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="map_file" default="$(find turtlebot3_navigation)/maps/map.yaml"/>
+  <arg name="open_rviz" default="false"/>
```

Comment:
What is this change about? I thought there was some change like this for API V2 that was needed, but that change wasn't committed before?

Reply:
open_rviz has always been false in the launch files in the world repo. The default value, which is “true”, is always overridden and never be used.

Reply:
Also, V2 requires it to be false

Reply:
see
https://github.com/aws-robotics/aws-robomaker-small-house-world/blob/ros1/launch/small_house_turtlebot_navigation.launch#L38
https://github.com/aws-robotics/aws-robomaker-small-house-world/blob/ros1/launch/turtlebot3_navigation.launch#L6
https://github.com/aws-robotics/aws-robomaker-bookstore-world/blob/ros1/launch/bookstore_turtlebot_navigation.launch#L41